lib/query.js

```diff
@@ -2,14 +2,18 @@
  * Functions to query properties of nodes
  */
 
+const {print} = require('math-parser')
+const build = require('./build')
+
 export const isIdentifier = node => node && node.type === 'Identifier'
 export const isApply = node => node && node.type === 'Apply'
+export const isParens = node => node && node.type === 'Parentheses'
 
 export const isOperation = node => isApply(node) && !isNumber(node)
 export const isFunction = node => isApply(node) && isIdentifier(node.op)
 
 // TODO: curry it?
-const _isOp = (op, node) => isApply(node) && node.op === op
+export const _isOp = (op, node) => isApply(node) && node.op === op
```

Comment:
Oh this was an accident. We could keep `_isOp` as it is since it seems to be working fine.